### PR TITLE
feat: adds command --help

### PR DESCRIPTION
**Pull request by dutterbutter**
**Changes introduced in this PR:**

- Adds `zksync-cli confirm-withdraw --help`
- Adds `zksync-cli create --help`
- Adds `zksync-cli deposit --help` 
- Adds `zksync-cli withdraw --help`

### CODE REVIEW

**Comment by uF4No** [comment]
let's wait to fix conflicts and merge until new version is released

**Comment by dutterbutter** [comment]
@uF4No updated to resolve conflicts.

File: bin/index.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
+#! /usr/bin/env node
```
Comment (MexicanAce) [inline bin/index.js:1]: Do we need this `bin/*.js` file?

File: src/deposit.ts [hunk 2]
```diff
@@ -5,12 +5,26 @@ import chalk from "chalk";
...
+  console.log(chalk.bold(`Options:`));
+  console.log(chalk.greenBright(`--l1-rpc-url=<URL>`));
+  console.log(`The URL of the L1 RPC provider.\n`);
+  console.log(chalk.greenBright(`--l2-rpc-url=<URL>`));
+  console.log(`The URL of the L2 RPC provider.\n`);
+}
+
+export default async function (zeek?: boolean, l1RpcUrl?: string, l2RpcUrl?: string) {
+
+  track("deposit", {zeek, network: "goerli"});
```
Comment (MexicanAce) [inline src/deposit.ts:25]: We should remove this `track("deposit", ...)` as we already do this at the end of the function

File: src/withdraw.ts [hunk 3]
```diff
@@ -4,12 +4,26 @@ import chalk from "chalk";
...
+  console.log(chalk.bold(`Options:`));
+  console.log(chalk.greenBright(`--l1-rpc-url=<URL>`));
+  console.log(`The URL of the L1 RPC provider.\n`);
+  console.log(chalk.greenBright(`--l2-rpc-url=<URL>`));
+  console.log(`The URL of the L2 RPC provider.\n`);
+}
+
+export default async function (zeek?: boolean, l1RpcUrl?: string, l2RpcUrl?: string) {
+
+  track("withdraw", {zeek, network: "goerli"})
```
Comment (MexicanAce) [inline src/withdraw.ts:24]: We should remove this extra `track("withdraw", ...)`

File: src/index.ts [hunk 4]
```diff
@@ -26,6 +26,7 @@ const availableOptions: string[] = [
 const option: string = process.argv[2];
 
 const main = async () => {
+  const helpFlag = Boolean(process.argv.filter((arg) => arg === "--help")[0]);
```
Comment (MexicanAce) [inline src/index.ts:29]: Should we also accept `-h`?
```suggestion
  const helpFlag = Boolean(process.argv.filter((arg) => ["--help", "-h"].includes(arg))[0]);
```

File: bin/index.js (hunk 1 above)
Comment (dutterbutter) [reply bin/index.js:1]: Nope adding to .gitignore

**Review by MexicanAce — APPROVED** [review]
LGTM 💯

**Pull request merged**